src/main/java/com/rackspace/ceres/app/config/AppProperties.java

```diff
@@ -33,49 +33,49 @@
 @Data
 @Validated
 public class AppProperties {
+
   @NotNull
   @DurationUnit(ChronoUnit.SECONDS)
   Duration rawTtl = Duration.ofHours(6);
 
   /**
-   * When initially creating the Cassandra data table schemas,
-   * this value will be used for the expired data garbage collection.
+   * When initially creating the Cassandra data table schemas, this value will be used for the
```

Comment:
Remove this unnecessary changes in comments.

Reply:
Fixed it